### PR TITLE
Upgrade caliburn.micro to 3.2.0

### PR DESCRIPTION
**Pull request by danielmarbach**
Connects to https://github.com/Particular/ServiceInsight/issues/774

### CODE REVIEW

File: src/ServiceInsight/ServiceInsight.csproj [hunk 1]
```diff
@@ -239,7 +240,7 @@
     <Reference Include="System.Transactions" />
     <Reference Include="System.Windows.Forms" />
     <Reference Include="System.Windows.Interactivity, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.SDK.Expression.Blend.1.0.0\lib\net45\System.Windows.Interactivity.dll</HintPath>
+      <HintPath>..\packages\Caliburn.Micro.3.2.0\lib\net45\System.Windows.Interactivity.dll</HintPath>
```
Comment (adamralph) [inline src/ServiceInsight/ServiceInsight.csproj]: Does this mean we can remove the installation of `Microsoft.SDK.Expression.Blend` that was added in #769?

**Comment by danielmarbach** [comment]
I did smoke test the tabs, the properties window and the paging yes

File: src/ServiceInsight/ServiceInsight.csproj (hunk 1 above)
Comment (danielmarbach) [reply src/ServiceInsight/ServiceInsight.csproj]: The expression blend SDK was removed automatically by nuget. I'm guessing yes.

**Comment by danielmarbach** [comment]
@HEskandari it wouldn't hurt. But I wonder why this has to be released just and why we can't accumulate a few more things on master and then release it together

**Comment by HEskandari** [comment]
For occasions we have more things coming through, we can delay the release a bit, but otherwise, we have 'release as soon as possible' rule of thumb.

**Comment by WilliamBZA** [comment]
Features or bugs should be released once ready. Things like this offer no tangible value to users so should not fall within the same rules as features or bugs.

**Pull request merged**